lib/src/over_react_redux/devtools/middleware.dart

```diff
@@ -0,0 +1,158 @@
+@JS()
+library over_react_redux.devtools.middleware;
+
+import 'dart:convert';
+import 'dart:html';
+import 'dart:js_util';
+
+import 'package:logging/logging.dart';
+import 'package:react/react_client/js_backed_map.dart';
+import 'package:redux/redux.dart';
+import 'package:redux_dev_tools/redux_dev_tools.dart';
+import 'package:js/js.dart';
```

Comment:
I'm just going to be an on an import kick this morning ;) 
```suggestion
import 'package:js/js.dart';
import 'package:logging/logging.dart';
import 'package:react/react_client/js_backed_map.dart';
import 'package:redux/redux.dart';
import 'package:redux_dev_tools/redux_dev_tools.dart';
```